src/components/gameScreen/gameScreen.container.js

```diff
@@ -115,41 +94,39 @@ export const GameScreenContainer = () => {
 
   const bookmarkSetter = async () => {
     try {
-      if (await AsyncStorageService.setBookmark({
-      question: currentQuestion,
-      help: filteredQuestion[0].help,
-      rightAnswer: filteredQuestion[0].rightAnswer,
-    })) setIsBookmarkSet(true);
-      else {setIsBookmarkSet(false)}
+      const isBookmarkSet = await AsyncStorageService.setBookmark({
+        question: store.currentQuestion.text,
+        help: store.currentQuestion.help,
+        rightAnswer: store.currentQuestion.rightAnswer,
+      })
+
+      isBookmarkSet ?
+        bookmarkStore.setIsBookmarkSet(true)
+        :
+        bookmarkStore.setIsBookmarkSet(false)
+
     } catch(e) {
       console.log('bookmark add error', e)
     }
   };
 
   const onNextQuestionButton = () => {
-    handleNextQuestion(currentScore, true)
+    handleNextQuestion( true)
```

Comment:
I think that you should consider linter configuration

Reply:
When I was reviewing test tasks for one company - code styling problems were one of the reasons I could not recommend a developer for a next interview.